docs/proposals/13-automated-seed-management.md

```diff
@@ -0,0 +1,93 @@
+# Automated Seed Management
+
+Automated seed management involves automating the different aspects of managing seeds in Garden clusters, such as:
+
+* Automatically adding or removing seeds as needed (auto-scaling).
+* Ensuring that the seeds capacity for shoots is not exceeded.
+* Migrating shoots away from unhealthy seeds, or from healthy but overloaded ones.
```

Comment:
TBH, I feel like each one of these items deserves its own GEP, as this is in sum a very huge topic.

I think, all 3 parts of this document are great enhancements and definitely make sense, but I FMPOV they should be more detailed in order to agree on a concrete implementation design (which is the main goal of these proposals anyways, see https://github.com/gardener/gardener/tree/master/docs/proposals).

So in order to allow more in-depth discussions, I would actually like to see 3 individual GEPs, each using the [GEP template](https://github.com/gardener/gardener/blob/master/docs/proposals/00-template.md).
WDYT?

Reply:
I am open to suggestions here. The current version of the GEP should be seen more as a starting point for a discussion, so if we together decide we need 3 different GEPs, I will split it.
I would personally keep them together in a single document, at least for the moment, because they are related and it should be easier to review and discuss. More details regarding the concrete design can still be added.